### PR TITLE
Add custom port description property for FEC.

### PR DESCRIPTION
**Pull request by wilmo119**
Reviewer: @poolakiran @kenchiang

### CODE REVIEW

File: openflow_input/bsn_forward_error_correction [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#version 5
+#version 6
+
+struct of_port_desc_prop_bsn_forward_error_correction : of_port_desc_prop_bsn {
+    uint16_t type == 0xffff;
+    uint16_t length;
+    uint32_t experimenter == 0x5c16c7;
+    uint32_t exp_type == 2;
+    uint32_t configured;
+    uint32_t enabled;
```
Comment (kenchiang) [inline openflow_input/bsn_forward_error_correction]: maybe add comments on the order of "nonzero if fec is configured" and "nonzero if fec is enabled"

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/pr-528-jenkins-loxigen_pull_req_artifacts-75

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_artifacts/75/

**Comment by bsn-abat** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.bigswitch.com/job/loxigen_pull_req_docker/82/

**Comment by wilmo119** [comment]
SELF-ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  Log file at http://vss1/abat/2016.11.30.1941-m.loxigen.master/abat.log

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
Log file at http://vss1/abat/2016.11.30.1941-m.loxigen.master/abat.log
/cc

**Pull request merged**